src/main/java/com/github/fishio/control/HelpScreenController.java

```diff
@@ -15,12 +22,7 @@
 public class HelpScreenController implements ScreenController {
 
 	private Log log = Log.getLogger();
-	
-	@FXML
-	private Label helpText;
-	
-	@FXML
-	private Label helpTextPowerups;
+	private HashMap<KeyCode, Vec2d> keymap = getKeyMap();
```

Comment:
The HashMap should be static (final), and should be initialized statically, since it never changes and is independent from the objects.

```java
private static final HashMap<KeyCode, Vec2d> keymap = new HashMap<>();
static {
    keymap.put(key, value);
    \\....
}
```

<organisation